### PR TITLE
Close classloaders

### PR DESCRIPTION
**Pull request by eatkins**

### CODE REVIEW

**Comment by eatkins** [comment]
This is in response to #4784.

File: main/src/main/scala/sbt/internal/SysProp.scala [hunk 1]
```diff
@@ -82,9 +82,11 @@ object SysProp {
    */
   lazy val color: Boolean = ConsoleAppender.formatEnabledInEnv
 
+  def closeClassLoaders: Boolean = getOrTrue("sbt.close.classloaders")
```
Comment (eed3si9n) [inline main/src/main/scala/sbt/internal/SysProp.scala]: I'd make this `sbt.classloader.close`.

**Review by eed3si9n — CHANGES_REQUESTED** [review]

File: main/src/main/scala/sbt/internal/SysProp.scala (hunk 1 above)
Comment (eed3si9n) [reply main/src/main/scala/sbt/internal/SysProp.scala]: https://github.com/sbt/sbt/pull/4787

Comment (eatkins) [reply main/src/main/scala/sbt/internal/SysProp.scala]: Done.

**Review by eed3si9n — APPROVED** [review]

**Pull request merged**